modules/update-service-uninstall-web-console.adoc

```diff
@@ -0,0 +1,16 @@
+[id="update-service-uninstall-web-console_{context}"]
+= Uninstalling the OpenShift Update Service Operator by using the web console
+
+You can use the {product-title} web console to uninstall the OpenShift Update Service Operator.
+
+.Prerequisites
+
+- All OpenShift Update Service applications have been deleted.
+
+.Procedure
+
+To uninstall the OpenShift Update Service Operator using the {product-title} web console:
+
+. In the {product-title} web console, click *Operators* -> *Installed Operators*.
+
+. Click *Uninstall Operator* from the menu of *OpenShift Update Service* from the list of installed Operators.
```

Comment:
A `Uninstall Operator` alert pops up, need to click `Uninstall` to proceed the uninstallation.
![Screenshot from 2021-03-25 15-54-49](https://user-images.githubusercontent.com/22511421/112438289-af39d580-8d82-11eb-9778-7929be4c3b69.png)

Reply:
Fixed